server/src/main/java/io/spine/server/model/DuplicateCommandHandlerError.java

```diff
@@ -51,22 +55,67 @@ private static String fmt(CommandHandlingClass duplicatingClass,
                               Map<Set<CommandClass>, CommandHandlingClass> registeredHandlers) {
         checkNotNull(duplicatingClass);
         checkNotNull(registeredHandlers);
+        @SuppressWarnings("MagicNumber") // the buffer size that should cover most cases.
         StringBuilder builder = new StringBuilder(512);
-        builder.append(format("The class %s declares handler methods for commands that are " +
-                              "already handled by other classes.", duplicatingClass));
+
+        builder.append(format("The class `%s` declares handler ", duplicatingClass));
+
+        // Do we have more than one command to report?
+        long totalDuplicatedCommands =
+                registeredHandlers.keySet()
+                                  .stream()
+                                  .mapToLong(Collection::size)
+                                  .sum();
+        checkState(totalDuplicatedCommands >= 1);
+        builder.append(
+                totalDuplicatedCommands == 1
+                ? "method for the command which is "
+                : "methods for commands that are "
+        );
+        builder.append("already handled by ");
+
+        // How many already handling classes do we have?
```

Comment:
It seems it better reads as `How many handling classes do we have already?` Not sure I get your original idea though.